pkg/terraform/module/handler.go

```diff
@@ -119,8 +119,7 @@ func needsUpdate(m *v1.Module) bool {
 func isPolling(spec v1.ModuleSpec) bool {
 	return len(spec.Content) == 0 &&
 		spec.Git.URL != "" &&
-		spec.Git.Commit == "" &&
-		spec.Git.Tag == ""
```

Comment:
missing &&

Reply:
no, I'm removing tag. Diff is quite confusing here, resulting code is
	return len(spec.Content) == 0 &&
		spec.Git.URL != "" &&
		spec.Git.Commit == ""